src/components/molecules/Button/styles.module.scss

```diff
@@ -0,0 +1,104 @@
+@import 'src/styles/variables';
```

Comment:
**[blocker]**

I think you don't need to have the "src/" part when importing styles. It should work with just "styles/variables"